S15-nfg/GraphemeBreakTest.t

```diff
@@ -1,6 +1,6 @@
 ## WHEN UPDATING UNICODE VERSION ALSO UPDATE docs/unicode-generated-tests.asciidoc
 use v6;
-my IO::Path $repo-dir      = $?FILE.IO.parent(2).add("3rdparty/Unicode/11.0.0/ucd/auxiliary/GraphemeBreakTest.txt");
+my IO::Path $repo-dir      = $?FILE.IO.parent(2).add("3rdparty/Unicode/13.0.0/ucd/auxiliary/GraphemeBreakTest.txt");
```

Comment:
Do we want to remove the version 11 GraphemeBreakTest.txt or is it still needed?

Reply:
Probably can be removed. It seems like the 13.0 file has all of the same info, just more.